README.md

```diff
@@ -5,17 +5,27 @@ Synchronizes a github repository with a local repository. Automatically deals wi
 
 # Installation
 
-If you don't use `pipsi`, you're missing out.
-Here are [installation instructions](https://github.com/mitsuhiko/pipsi#readme).
+Before installing, make sure you have a **Python 3** installation with
+the Jupyter environment installed.
 
-Simply run:
+To install gitautosync, first install the package with `pip`:
 
-    $ pipsi install .
+    pip install gitautosync
```

Comment:
We should tell people to install from git for now. 

pip install git+https://github.com/data-8/gitautosync